src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -135,6 +135,38 @@ private void OnDispatchedToUI(object sender, UIDispatcherEventArgs e)
         /// </summary>
         public event Action<PortModel> PortDisconnected;
 
+        public class NodeExecutionEventArgs : EventArgs
```

Comment:
all the public properties and the class itself should have some ///summary tags please.